### PR TITLE
Update network-plugins.md

### PR DESCRIPTION
**Pull request by jakowicz**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ By default if no kubelet network plugin is specified, the `noop` plugin is used,
 
 The CNI plugin is selected by passing Kubelet the `--network-plugin=cni` command-line option.  Kubelet reads a file from `--cni-conf-dir` (default `/etc/cni/net.d`) and uses the CNI configuration from that file to set up each pod's network.  The CNI configuration file must match the [CNI specification](https://github.com/containernetworking/cni/blob/master/SPEC.md#network-configuration), and any required CNI plugins referenced by the configuration must be present in `--cni-bin-dir` (default `/opt/cni/bin`).
 
-If there are multiple CNI configuration files in the directory, the first one in lexicographic order of file name is used.
+If there are multiple CNI configuration files in the directory, the first by name in lexicographicorder is used.
```
Comment (miteshskj) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: missing space in "lexicographic order"

Comment (sftim) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: Here's a suggested reword
```suggestion
If there are multiple CNI configuration files in the directory, the kubelet uses the configuration file that comes first by name in lexicographic order.
```

**Comment by bradtopol** [comment]
Assigning @sftim since he is already engaged on this. Tim, do you feel you need a technical reviewer as well? I can find one if need be.
/assign @sftim

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md (hunk 1 above)
Comment (jakowicz) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: @miteshskj Thanks that was hopeless of me
@sftim That's actually much better

**Pull request merged**